src/main/java/com/slimebot/games/Game.java

```diff
@@ -0,0 +1,82 @@
+package com.slimebot.games;
+
+import com.slimebot.main.Main;
+import net.dv8tion.jda.api.entities.channel.middleman.MessageChannel;
+import net.dv8tion.jda.api.hooks.ListenerAdapter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+
+public abstract class Game extends ListenerAdapter {
+
+    public final UUID uuid;
+    public final long channelId;
+    public final long guildId;
+    public long gameMaster;
+    public List<Long> player;
+    public GameStatus status;
+
+    public Game(long gameMaster, long channelId, long guildId) {
+        this.gameMaster = gameMaster;
+        this.channelId = channelId;
+        this.guildId = guildId;
+        this.uuid = UUID.randomUUID();
+
+        player = new ArrayList<>();
+        player.add(gameMaster);
+
+        PlayerGameState.setGameState(gameMaster, new PlayerGameState(this));
+
+        status = GameStatus.WAITING;
+
+        Main.jdaInstance.addEventListener(this);
+
+        // End the game if its 15min WAITING
+        Main.executor.schedule(() -> {
+            if (status == GameStatus.WAITING) end();
```

Comment:
No space between `if` and `(`. I'm not going to comment this everywhere, just change your intellij style and do a code reformat